### ISSUE TITLE
Set Hummingbird as the default theme

### ISSUE THREAD

**Issue by ibahloul-ps** [PrestaShop/PrestaShop#37480]
Modify the installation process so that the default theme is now Hummingbird
This requires a bit of investigation about all the places that handle themes and where the default theme is defined

A quick search returned these two places:
- https://github.com/PrestaShop/PrestaShop/blob/b548cd5c389c5b61e625b485b0e3613c178ed21d/install-dev/controllers/http/process.php#L107
- https://github.com/PrestaShop/PrestaShop/blob/f352ea7f456a668c07594cfc12976598ded64949/install-dev/init.php#L130

But there may be others.

### First we need to homogenize the defult theme management (one PR)

- this issue https://github.com/PrestaShop/PrestaShop/issues/35299 lists all the places impacted more accurately, but may not be exhaustive
- We need one source of truth that impact all these references:
  - careful the legacy context uses PHP const and is not initialised the same way
  - we need to be able to modify the default theme during install in both browser AND cli install
  - we should handle an environment variable to change this source of more dynamically (again impact on installation)
- The environement approach will also help us with the UI tests migration, because we could dynamically choose which default theme to use for each campaign (by adding a new value in the matrix during the migration phase)

### Activating hummingbird as default for the beta

- Once we handled this homogenization we can handle all the release process for the beta, and we still have all our tests backing us up.
- Once we've done everything the last step will be a PR that switches `classic` to `hummingbird` in a single place (source of truth), this PR can target a beta build branch, this way we don't "poullte" the 9.1.x branch
- We will merge this on the build branch knowing that many UI tests will break, and probably also some of the PR's CIs, ideally if we could have at least the CI green it would be great

## Warnings

- in browser install check if hummingbird is correctly checked, if the list is hard coded put it in first
- after install check the generated image_types they must be hummingbord's ones (check the dimensions)

**Comment by venditdevs** [comment]
Considering the laws in Europe regarding abuse of dominant market positions, i strongly advise to research the potential negative impact this could have.

There are a lot of companies which recieved high fines for installing a own product or pull along an own solution without having a choice for a alternative.

https://ec.europa.eu/commission/presscorner/detail/en/memo_17_1785
https://ec.europa.eu/commission/presscorner/detail/en/memo_07_359
https://inhouse-legal.eu/antitrust-competition-law/another-eu-antitrust-action-against-microsoft-a-new-chapter-in-a-longstanding-saga/

**Comment by tleon** [comment]
We will do that as one of the last step of the release. As seen with the team.

**Comment by jolelievre** [comment]
Indeed, we need to handle all the release process first, we can't do that effectively if we break all our test because of the default theme
